Core1Platform/UserInterface/01_ProfileMenu.md

```diff
@@ -0,0 +1,44 @@
+# Profile menu
+
+*Any workspace > Avatar icon*
```

Comment:
Ich würde einfach "Workspace" schreiben, auch in den anderen Pfaden. Wenn man es so generell angibt, würde man alle/jedes meinen, oder?

Reply:
ist auch jeder Workspace